nuitka/BytecodeCaching.py

```diff
@@ -83,24 +84,29 @@ def getCachedImportedModulesNames(module_name, source_code):
     if data["module_name"] != module_name:
         return None
 
-    return [
-        (ModuleName(used_module_name), line_number)
-        for (used_module_name, line_number) in data["modules_used"]
-    ]
+    used_modules = list(data["modules_used"])
```

Comment:
Why would we convert to a list here, elements of a tuple can be modified still, if it's their properties only